tests/src/Unit/OgPermissionHandlerTest.php

```diff
@@ -91,9 +91,11 @@ protected function mockModuleExtension($module, $name) {
    * @covers ::moduleProvidesPermissions
    */
   public function testBuildPermissionsYaml() {
-    // defining OG_ANONYMOUS_ROLE on the fly since OG isn't enabled for this
-    // test and that could cause for a test's failure.
+    // Define the OG role constants since OG isn't enabled, however
+    // \Drupal\og\OgPermissionHandler::buildPermissionsYaml is using it.
     define('OG_ANONYMOUS_ROLE', 'non-member');
+    define('OG_AUTHENTICATED_ROLE', 'member');
```

Comment:
This is still really wrong. We need these constants to be on a class like Og so they can be autoloader.  At the very least we will need to check if the constant exists before defining it.